src/histogram_merging.jl

```diff
@@ -175,7 +179,9 @@ function fit_histogram_merging(ys::Array{Float64, 1}, sigma::Float64, num_target
 
     # Select the num_holdout_pieces'th largest error (counting from the largest
     # error down) as threshold.
-    error_threshold = select(candidate_errors, max(0, length(candidate_pieces) - num_holdout_pieces + 1))
+    sorted_errors = sort(candidate_errors)
```

Comment:
Is there no select in Julia any more? IIRC the select is O(n) while sort is O(n log n), but I may misremember.

Reply:
Thank you so much for your feedback. You are right about the performance difference between sort (O(n log n)-time) and select (O(n)-time). Using sort could indeed worsen performance.

The select function was renamed to `partialsort` in this [PR](https://github.com/JuliaLang/julia/pull/23051/files) because [one person reported the name was misleading](https://github.com/JuliaLang/julia/issues/22791).

I will use `partialsort` as follows:
```julia
error_threshold = partialsort(candidate_errors, max(0, length(candidate_pieces) - num_holdout_pieces + 1))
```